### PR TITLE
Changed nightly branch logo

### PR DESCRIPTION
**Pull request by ZhichGaming**
This PR changes the logo on the develop branch from the stable PlayCover logo to the nightly logo.

### CODE REVIEW

**Comment by Wind-Explorer** [comment]
Please double-check that you are not attempting to change the logo of the release version. As far as I can tell the image altered is in `images/logo.png` which is where README.md and other regular modules source the image from.

**Comment by zanderp25** [comment]
I guess this'll work, it'll just need to be explicitly excluded from the version updates.

**Review by zanderp25 — APPROVED** [review]

**Comment by lixin9311** [comment]
I think it is better to keep the 2 sets of logos in the code base at the same time.
Then we specify the logo at build time by using environment variables such as `CHANNEL=nightly`
Some FYI <https://www.raywenderlich.com/1716-how-to-change-your-app-icon-at-build-time>

**Comment by ZhichGaming** [comment]
I do not know how to do the thing that you linked, but I tried this: https://engineering.circle.com/different-app-icons-for-your-ios-beta-dev-and-release-builds-af4d209cdbfd. Unfortunately, it did not work since it probably was something iOS only or something to the sort. I still have added the nightly and stable icons though I discarded my other changes.

**Comment by lixin9311** [comment]
No worry, I will cherry pick later.

**Comment by zanderp25** [comment]
whoops

**Comment by ZhichGaming** [comment]
I've banged my head trying to solve this, and I was just made to realize by Wind that I have already solved it... We did not have to do any complicated things like environment variables, we just had to select the freaking icon to build with. 
Screen Shot 2022-08-24 at 23 51 21

File: PlayCover.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -781,14 +781,14 @@
...
-				CODE_SIGN_IDENTITY = "Mac Developer";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 36;
 				DEAD_CODE_STRIPPING = YES;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				DEVELOPMENT_ASSET_PATHS = "PlayCover/Preview\\ Content";
-				DEVELOPMENT_TEAM = 792V9HMJW3;
```
Comment (Wind-Explorer) [inline PlayCover.xcodeproj/project.pbxproj]: Please unstage the chunks related to the change of signing certificate.

**Review by Wind-Explorer — CHANGES_REQUESTED** [review]

File: PlayCover.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ZhichGaming) [reply PlayCover.xcodeproj/project.pbxproj]: Done, I don't know how this got commited, I apologize.

**Comment by ZhichGaming** [comment]
I noticed that the dev icons have changed from gray to orange/yellow, has that been decided by people already that we will be using that?

**Comment by ZhichGaming** [comment]
You can merge this now, I've fixed the conflicts.

**Comment by IsaacMarovitz** [comment]
This can not be merged until these icons are made to work with the automated build pipelines.

**Comment by ZhichGaming** [comment]
I've added a nightly scheme that uses the nightly icons but I do not know how to change the fastfile for it to run that scheme in only nightly builds.

**Comment by ZhichGaming** [comment]
I've added a nightly lane to fastfile but I am not experienced with fastlane at all so please review it carefully before merging.

**Comment by ZhichGaming** [comment]
Now, it is finally ready for review.

**Comment by IsaacMarovitz** [comment]
Can confirm, works locally

**Review by IsaacMarovitz — APPROVED** [review]

**Review by Depal1 — APPROVED** [review]

**Pull request merged**